for debugger to work, remove multiprocessing i.e (num_workers=4)
The error is as following: Couldn't find a debug adapter descriptor for debug type 'Python Kernel Debug Adapter' (extension might have failed to activate)